README.md

```diff
@@ -18,7 +18,7 @@ Install as usual, see [this](https://drupal.org/documentation/install/modules-th
 
 Configure the path patterns using the usual Pathauto interface, at Administration » Configuration » Search and Metadata » URL Aliases » Patterns (admin/config/search/path/patterns). 
 
-![Pathauto Configuration](https://raw.githubusercontent.com/wiki/Islandora/islandora_pathauto/images/sample-pathauto-configuration.png)
+![pathautoconfiguration](https://user-images.githubusercontent.com/877577/47673120-fbac6700-db70-11e8-98fc-51f5b2cb8555.png)
```

Comment:
Awesome job updating the README! One thing to note is that the text between the brackets, despite not displaying, still serves as alt text which screenreaders use, so its a best practice to make sure that this text is readable. For this reason I'd recommend going with the new URL to the new image, but keeping the bracketed text `Pathauto Configuration`.